libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -160,6 +117,12 @@ bool AC_AutoTune::init_internals(bool _use_poshold,
         FALLTHROUGH;
 
     case UNINITIALISED:
+        // initializes dwell test sequence for rate_p_up and rate_d_up tests for tradheli
+        freq_cnt = 0;
```

Comment:
I wonder if this initialisation should instead be put into a heli specific version of "backup_gains_and_initialise".  So I'm thinking heli implements backup_gains_and_initialise() and within it's implementation it does this extra stuff and calls the parent AC_AutoTune::backup_gains_and_initialise().

Reply:
done